src/IB/IBFEPatchRecoveryPostProcessor.cpp

```diff
@@ -414,9 +414,7 @@ IBFEPatchRecoveryPostProcessor::initializeCauchyStressSystem()
     {
         for (unsigned int j = i; j < NDIM; ++j)
         {
-            std::ostringstream os;
-            os << "sigma_" << (i == 0 ? "x" : i == 1 ? "y" : "z") << (j == 0 ? "x" : j == 1 ? "y" : "z");
-            sigma_system->add_variable(os.str(), d_interp_order, LAGRANGE);
+            sigma_system->add_variable("sigma_" + (i == 0 ? "x" : i == 1 ? "y" : "z") + (j == 0 ? "x" : j == 1 ? "y" : "z"), d_interp_order, LAGRANGE);
```

Comment:
This is too complicated for an inline ternary expression and I am surprised that it works. Could you do something like
```cpp
std::string var_name = "sigma_";
var_name += (i == 0 ? 'x') : i == 1 ? 'y' : 'z');
var_name += (j == 0 ? 'x') : j == 1 ? 'y' : 'z');
```